### PR TITLE
updates copy and steps

### PR DESCRIPTION
**Pull request by csell5**

### CODE REVIEW

File: src/routes/activities/components/form/ActivityForm.svelte [hunk 1]
```diff
@@ -227,7 +227,7 @@
             <SectionTitle
               title="Activity Attributes"
               stepNumber="3"
-              description="It's meta time. Let's add some attributes about this activity to best help us sort, and help geeks find it." />
+              description="It's meta data time. Let's add some attributes about this activity to help us best sort and help categorize it." />
```
Comment (brettski) [inline src/routes/activities/components/form/ActivityForm.svelte:230]: Sugested edit:
"It's meta data time. Let's add some attributes about this activity to help us best sort and categorize it."

File: src/routes/activities/components/form/_Resources.svelte [hunk 2]
```diff
@@ -32,8 +33,9 @@
       </label>
       <div>
         <p class="mt-1 text-sm text-gray-400">
-          Here is where you can provide pointers to any applicable resources,
-          your presentation, etc. You will be able to this list later.
+          Please provide links to any helpful resources, your presentation,
+          SDK's, that someone might want after attending. You will be able to
+          this list later.
```
Comment (brettski) [inline src/routes/activities/components/form/_Resources.svelte:38]: "You will be able to this list later."   =>
"You will be able to add to this list later."

File: src/routes/activities/components/form/_WorkshopDetails.svelte [hunk 3]
```diff
@@ -74,7 +74,11 @@
         ></span>
       </div>
       <div>
-        <p class="mt-1 text-sm text-gray-400">WRITE ME</p>
+        <p class="mt-1 text-sm text-gray-400">
+          Please break down the overall flow of this workshop. We recognize
+          things will likely change, but help every understand in detail what
```
Comment (brettski) [inline src/routes/activities/components/form/_WorkshopDetails.svelte:79]: "...but help every ..." 
Not sure what you were getting at with the every there.

**Review by brettski — CHANGES_REQUESTED** [review]

File: src/routes/activities/components/form/_WorkshopDetails.svelte (hunk 3 above)
Comment (csell5) [reply src/routes/activities/components/form/_WorkshopDetails.svelte:79]: I meant to say, everyone.

File: src/routes/activities/components/form/_WorkshopDetails.svelte [hunk 4]
```diff
@@ -23,9 +23,9 @@
           class="absolute top-0 left-0 transform -translate-x-4 -translate-y-4 block h-2 w-2 rounded-full bg-red-400"
         ></span>
       </div>
-      <div>
+      <!-- <div>
```
Comment (brettski) [inline src/routes/activities/components/form/_WorkshopDetails.svelte:26]: Why not delete it?

**Review by brettski — APPROVED** [review]

File: src/routes/activities/components/form/_WorkshopDetails.svelte (hunk 4 above)
Comment (csell5) [reply src/routes/activities/components/form/_WorkshopDetails.svelte:26]: because I figured we'd add something later when I was more creative.

**Pull request merged**